lib/coverband/configuration.rb

```diff
@@ -222,8 +222,8 @@ def search_paths=(path_array)
     # Don't allow the ignore to override things like gem tracking
     ###
     def ignore=(ignored_array)
-      ignored_array.map { |ignore_str| Regexp.new(ignore_str) }
-      @ignore = (@ignore + ignored_array).uniq
+      ignored_array = ignored_array.map { |ignore_str| Regexp.new(ignore_str) }
+      @ignore |= ignored_array
```

Comment:
so I think we want to assign here with a uniq... the purpose of this is that folks can add overrides to ignores... traditionally those have always been strings... so it is good to fix the bug, but I think we still need the `(@ignore + ignored_array).uniq` type logic.

Reply:
`@ignore |= ignored_array` is the same as `@ignore = (@ignore + ignored_array).uniq`. 
Correct me if I am wrong.

Reply:
ha you are correct, I just don't use the single `|=` very often... this works